django_telegrambot/apps.py

```diff
@@ -43,42 +45,58 @@ class DjangoTelegramBot(AppConfig):
     name = 'django_telegrambot'
     verbose_name = 'Django TelegramBot'
     ready_run = False
-    bot_tokens = []
-    bot_usernames = []
-    dispatchers = []
-    bots = []
-    updaters = []
+    bots_data = OrderedDict()
     __used_tokens = set()
 
     @classproperty
     def dispatcher(cls):
-        #print("Getting value default dispatcher")
-        cls.__used_tokens.add(cls.bot_tokens[0])
-        return cls.dispatchers[0]
+        try:
+            #print("Getting value default dispatcher")
+            bot_data = list(cls.bots_data.values())[0]
+            cls.__used_tokens.add(bot_data['token'])
+            return next(bot_data['dispatcher'])
+        except StopIteration:
+            raise ReferenceError("No bots are defined")
 
     @classproperty
     def updater(cls):
-        #print("Getting value default updater")
-        cls.__used_tokens.add(cls.bot_tokens[0])
-        return cls.updaters[0]
+        try:
+            #print("Getting value default dispatcher")
+            bot_data = list(cls.bots_data.values())[0]
+            cls.__used_tokens.add(bot_data['token'])
+            return next(bot_data['updater'])
+        except StopIteration:
+            raise ReferenceError("No bots are defined")
+
 
     @classmethod
-    def get_dispatcher(cls, bot_id=None, safe=True):
+    def getBotById(cls, bot_id=None, safe=True):
```

Comment:
I would prefer to stick to scnake_case style: `get_bot_by_id()` but however, having wrappers for compatibility is even better..

Reply:
This function actually should be a private function I guess, because no one would need the data dict, and even if they needed, they'd read the config from the settings.